.gitignore

```diff
@@ -4,3 +4,5 @@ build/
 /metrictank
 /cmd/mt-split-metrics-by-ttl/mt-split-metrics-by-ttl
 /cmd/mt-store-cat/mt-store-cat
+/cmd/mt-whisper-importer-reader/main
+/cmd/mt-whisper-importer-writer/main
```

Comment:
interesting, I just run `go build`, and then it names the binary `mt-whisper-importer-writer`, but if you run it like you do, it's called `main` indeed. I think you should just run `go build` (this also helps if there are other go files needed to compile, it'll use everything that is imported by main), and add the "real" names to gitignore.